ldms/src/sampler/zfs_topvdevs/Makefile.am

```diff
@@ -0,0 +1,21 @@
+pkglib_LTLIBRARIES =
+lib_LTLIBRARIES =
+dist_man7_MANS =
+dist_man1_MANS =
+
+AUTOMAKE_OPTIONS = subdir-objects
+
+AM_CPPFLAGS = @OVIS_INCLUDE_ABS@ \
+	      -I/usr/include/libspl \
+	      -I/usr/include/libzfs
+
+AM_LDFLAGS = @OVIS_LIB_ABS@
+COMMON_LIBADD = -lsampler_base -lldms -lovis_util -lcoll \
+		@LDFLAGS_GETTIME@
+
+if ENABLE_PROCNETDEV
```

Comment:
Remove this "if ENABLE_PROCNETDEV" and the associated endif. This needs to be done in all three Makefile.am. The conditional build of these samplers is handled one directory up, so no conditional is needed here at all.